### ISSUE TITLE
Plot a graph for sniffed packet

### ISSUE THREAD

**Issue by Anshu-Net** [buckyroberts/Python-Packet-Sniffer#3]
I am trying to plot a graph of sniffed packets.
My idea is to plot a graph for number of different protocol used every minute.
A graph which show in every minute number of  UDP, TCP, ICMP, IPV4 protocol is used.
can you help me with importing this outputs and plot a graph.

**Comment by techyringo** [comment]
Yes i can plot bipartite graph between source and destination and also mobi patent graph..do you need it???
http://bl.ocks.org/mbostock/1153292 like this
